api/src/main/java/org/eclipse/microprofile/concurrent/ManagedExecutorConfig.java

```diff
@@ -164,4 +166,47 @@
      * <code>maxQueued</code> value is 0 or less than -1.
      */
     int maxQueued() default -1;
+
+        /**
+        * Util class used for inline creation of {@link ManagedExecutorConfig} annotation instances.
+        */
+        public final class ManagedExecutorConfigLiteral extends AnnotationLiteral<ManagedExecutorConfig> implements ManagedExecutorConfig {
```

Comment:
I think the class name can be reduced to just `Literal` on all of these, since repeating the class name is redundant: `ManagedExecutorConfig.ManagedExecutorConfigLiteral`.  Using just `Literal` would also be consistent with the CDI spec literal naming convention

Reply:
same comment also applies to the other literal classes introduced in this PR

Reply:
Within CDI spec there are both variations, mostly depending on whether CDI owns that annotation or not. Ultimately it is a matter of choice. I am fine with just `Literal`.